src/components/Navbar/links.tsx

```diff
@@ -10,6 +10,7 @@ export const navLinks: NavLink[] = [
   { title: "Events", slug: "events" },
   { title: "Visit Us", slug: "visit" },
   { title: "Contact Us", slug: "contact" },
+  { title: "Profile Page", slug: "profile" },
```

Comment:
This is fine, but this causes the profile page header to have two lines, try to make this into one line